src/mono/sample/wasm/console-node-ts/Wasm.Console.Node.TS.Sample.csproj

```diff
@@ -1,15 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <WasmCopyAppZipToHelixTestDir>false</WasmCopyAppZipToHelixTestDir>
-    <WasmMainJSPath>main.ts</WasmMainJSPath>
-    <!--
-    <WasmEnableES6>true</WasmEnableES6>
-    -->
+    <!-- Workaround before https://github.com/dotnet/runtime/issues/63465 is solved. -->
+    <WasmMainJSPath>tsconfig.json</WasmMainJSPath>
   </PropertyGroup>
 
-  <Target Name="TypeScript" AfterTargets="WasmBuildApp" DependsOnTargets="Build">
+  <!-- 
+    We need to install typescript definition (so tsc knows runtime types) and package.json (tsc knows about runtime files) from artifacts directory.
+  -->
+  <Target Name="CopyRelinkedPackage" AfterTargets="WasmBuildApp" DependsOnTargets="Build" Inputs="
+          $(MicrosoftNetCoreAppRuntimePackNativeDir)/dotnet.d.ts;
+          $(MicrosoftNetCoreAppRuntimePackNativeDir)/package.json;" Outputs="bin/$(Configuration)/AppBundle/.npm-stamp">
+    <ItemGroup>
+      <NpmPackageFiles Include="$(MicrosoftNetCoreAppRuntimePackNativeDir)/dotnet.d.ts"/>
+      <NpmPackageFiles Include="$(MicrosoftNetCoreAppRuntimePackNativeDir)/package.json"/>
```

Comment:
nit: You could move this outside the target, and then use the item reference in the `Inputs` too.